src/ui/ViewModels/Enums/CourseVariantStatus.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace GovUk.Education.ManageCourses.Ui.ViewModels.Enums
+{
+    public enum CourseVariantStatus
```

Comment:
Why can't we re-use CourseRunningStatus? Seems to be the same values. Does this have a different meaning?

Reply:
merge issues file was renamed and moved and had an extra enum added